src/ui/marker.js

```diff
@@ -36,6 +37,7 @@ type Options = {
  * @param {string} [options.color='#3FB1CE'] The color to use for the default marker if options.element is not provided. The default is light blue.
  * @param {number} [options.scale=1] The scale to use for the default marker if options.element is not provided. The default scale corresponds to a height of `41px` and a width of `27px`.
  * @param {boolean} [options.draggable=false] A boolean indicating whether or not a marker is able to be dragged to a new position on the map.
+ * @param {?number} [options.clickTolerance=null] A number specifying the click tolerance in pixels. For draggable markers, movement must exceed this tolerance to be treated as a drag. A `null` value inherits click tolerance from `Map`.
```

Comment:
What is click tolerance here? Is it pixels away from the marker that are still counted as clicking the marker or is a small drag while holding the mouse down within the tolerance is still counted as a click and not a drag?

They are two different things and I can see users getting confused which one this affects.

Reply:
I see from the changelog entre this would be the latter. I think we should make this clearer in the description to avoid people getting confused it's the former.

Reply:
Yes, that is a good clarification. I do still think it should be called `clickTolerance` because that is the name used for the `Map` option. (The ambiguity you allude to doesn't really exist for the map, though.) 

I'll change the jsdoc description here to mirror the map description more closely:

> The max number of pixels a user can shift the mouse pointer during a click for it to be considered a valid click (as opposed to a mouse drag).